docs/resiliency/index.asciidoc

```diff
@@ -89,6 +89,19 @@ Further issues remain with the retry mechanism:
 See {GIT}9967[#9967]. (STATUS: ONGOING)
 
 [float]
+=== Wait on incoming joins before electing local node as master (STATUS: ONGOING)
+
+During master election each node pings in order to discover other nodes and validate the liveness of existing
+nodes. Based on this information the node either discovers an existing master or, if enough nodes are found
+(see <<master-election,`discovery.zen.minimum_master_nodes>>) a new master will be elected. Currently, the node that is
```

Comment:
the `master-election` ID is in the ref docs, not in the resiliency page.  you need an absolute link for that to work.